Owncloud iOs Client/Files/PreviewSupportedFiles/Office/OfficeFileView.m

```diff
@@ -71,7 +71,7 @@ - (void)setFrame:(CGRect)frame {
  * Method to go back in the navigation
  */
 - (void)goBack{
-    
```

Comment:
Please, could you edit the files with the same editor that you did the first time. Now all tabulations in the files are detected as new lines. Thanks @laurentperrinet